Add filtering based off of "active" query param in Web UI

wbh1 opened the pull request:
Closes: #1867 

This is my first foray into Elm, so please let me know if I missed anything :)

@stuartnelson3 -- I'm not sure who to add as a reviewer, so just tagging you.

Thanks!

stuartnelson3 commented on ui/app/README.md:
````diff
@@ -13,6 +13,11 @@ query string containing the following will additionally show silenced alerts.
 
+In order to to show _only_ silenced alerts, update the query string to hide active alerts.
+```
+http://alertmanager/#/alerts?silenced=true&active=false
````
👌

stuartnelson3 commented on ui/app/src/Views/AlertList/Views.elm:
```diff
@@ -20,13 +20,13 @@ import Views.ReceiverBar.Views as ReceiverBar
                 [ type_ "checkbox"
                 , class "custom-control-input"
-                , checked (Maybe.withDefault False maybeShowSilenced)
+                , checked (Maybe.withDefault False maybeChecked)
```
I was just looking at this and thinking about that param name, thanks for fixing this!

wbh1 commented:
> Looks awesome!
> 
> Do you want to add a test for `generateQueryString` to show that setting `showActive` to `Just False` results in `active=false` in the query string?

I've added the test for this :) I also went ahead and removed one of the tests that was basically a duplicate. 

```elm
test "should default silenced & inhibited parameters to false and active to true if showSilenced is Nothing" <|
            \() ->
                Expect.equal "?silenced=false&inhibited=false&active=true"
                    (Utils.Filter.generateQueryString { receiver = Nothing, group = Nothing, customGrouping = False, text = Nothing, showSilenced = Nothing, showInhibited = Nothing, showActive = Nothing })
```
tested the same thing as 
```elm
test "should not render keys with Nothing value except the silenced, inhibited, and active parameters, which default to false, false, true, respectively." <|
            \() ->
                Expect.equal "?silenced=false&inhibited=false&active=true"
                    (Utils.Filter.generateQueryString { receiver = Nothing, group = Nothing, customGrouping = False, text = Nothing, showSilenced = Nothing, showInhibited = Nothing, showActive = Nothing })
```

stuartnelson3 commented:
Thanks!
